docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source.md

```diff
@@ -0,0 +1,107 @@
+
+## Setup
+
+### Vendor configuration
+
+:::note
+The threat intel vendor must follow the STIX/TAXII 2.0 or 2.1 standard.
+:::
+
+1. Identify a vendor who uses the STIX/TAXII 2.0 or 2.1 standard for sharing threat intelligence indicators
+1. Follow their documentation to obtain the following information:
+  * The STIX/TAXII version they provide (either 2.0 or 2.1)
+  * The TAXII Discovery URL
+  * Your authentication credentials if required
+
+
+### Source configuration
+
+When you create an TAXII 2 Client Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure an TAXII 2 Client Source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **TAXII 2 Client**.
+1. Enter a **Name** for the Source. The description is optional.
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. Select the STIX/TAXII version the vendor is using (2.0 or 2.1).
+1. Enter the TAXII Discovery URL provide by the vendor.
+1. If HTTP basic authentication is required, check **Use Basic Auth** and provide your vendor username and password.
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON Configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

Comment:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

Reply:
I know you were just following the template here, but I like a simple link to the article better.